.github/ISSUE_TEMPLATE/nightly_build_failure.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Nightly build failure report
+about: Lightweight incident report for nightly build failure
+title: Nightly Build Failure XXXX-XX-XX
+labels:
+  - bug
+  - nightly-builds
+assignees: catalyst-cooperative/inframundo
+---
+
+# Overview
+
+What do you think happened, if anything?
+
+# Next steps
+
+What next steps do we need to do to understand or remediate the issue?
+
+# Verify that everything is fixed!
```

Comment:
This would be the night after, after pushing the fix? Or after kicking off another build?